src/ucp/core/ucp_ep.inl

```diff
@@ -283,4 +283,10 @@ static UCS_F_ALWAYS_INLINE int ucp_ep_use_indirect_id(ucp_ep_h ep)
     return ep->flags & UCP_EP_FLAG_INDIRECT_ID;
 }
 
+static UCS_F_ALWAYS_INLINE int
+ucp_ep_config_err_mode_eq(ucp_ep_h ep, ucp_err_handling_mode_t err_mode)
+{
+    return ucp_ep_config(ep)->key.err_mode == err_mode;
+}
```

Comment:
why is new function needed? we already have such one

Reply:
it's code common for `ucp_ep_config_err_mode_check_mismatch()` and new code in `ucp_proto_request_pack_rkey()`

Reply:
maybe just extend old one like this?
```c
ucs_status_t
ucp_ep_config_err_mode_check_mismatch(ucp_ep_h ep,
                                      ucp_err_handling_mode_t err_mode,
                                      const char* err_message)
```

Reply:
we decided to use inline function in ucp_proto_request_pack_rkey(), its fast-path